Backends/Graphics4/OpenGL/Sources/Kore/VertexBufferImpl.cpp

```diff
@@ -60,6 +60,10 @@ Graphics4::VertexBuffer::VertexBuffer(int vertexCount, const VertexStructure& st
 
 	glGenBuffers(1, &bufferId);
 	glCheckErrors();
+	glBindBuffer(GL_ARRAY_BUFFER, bufferId);
+	glCheckErrors();
+	glBufferData(GL_ARRAY_BUFFER, myStride * myCount, nullptr, this->usage);
```

Comment:
No need to copy the data the first time, correct?
I used "this->usage" since it was used in the switch above, but if it's preferable to give it another name to avoid problems in the future I can change it to usageGL or usageMode.